src/EA.Weee.Web/Scripts/weee-application.js

```diff
@@ -109,8 +109,12 @@
         locale: "en"
 	    }
     );
+
+    $('.govuk-tabs__tab--selected').focus();
 });
```

Comment:
Good one. i think the only downside is even when the page loads it will get highlighted

Reply:
Which I think it should... looks like AATF tab that is initially loaded doesnt get the selected style... think ill add that